### PR TITLE
UI: Portfolio project

### PR DESCRIPTION
**Pull request by austin-pham-goldenowl**

### CODE REVIEW

File: src/pages/Portfolio/index.js [hunk 1]
```diff
@@ -50,14 +53,16 @@ const Services = () => (
...
                   </div>
-                </div>
+                </Link>
+              </div>
               ))}
             </div>
           </Tab>
           <Tab eventKey="web" title="Web development">
             <div className="row">
               {[...tabContent].reverse().map((item, index) => (
```
Comment (hansanwok) [inline src/pages/Portfolio/index.js]: why need ...

File: src/pages/Portfolio/index.js [hunk 2]
```diff
@@ -74,47 +79,43 @@ const Services = () => (
...
-                    }}
-                  >
-                    <h4
+                  <Link to={`/portfolio${item.url}`}>
+                    <div
+                      className="portfolio__card"
                       style={{
-                        color: item.isWhite ? "#fff" : "#3e403d"
+                        marginLeft: index % 2 === 0 ? "auto" : 0,
+                        marginLeft: index % 2 === 0 ? 0 : "auto",
```
Comment (hansanwok) [inline src/pages/Portfolio/index.js]: same condition but diff in result?

File: src/pages/Portfolio/index.js [hunk 3]
```diff
@@ -34,6 +36,7 @@ const Services = () => (
             <div className="row">
               {tabContent.map((item, index) => (
                 <div className="col-md-6">
```
Comment (hansanwok) [inline src/pages/Portfolio/index.js]: where is key?

File: src/pages/Portfolio/index.js [hunk 4]
```diff
@@ -50,14 +53,16 @@ const Services = () => (
...
-                </div>
+                </Link>
+              </div>
               ))}
             </div>
           </Tab>
           <Tab eventKey="web" title="Web development">
             <div className="row">
               {[...tabContent].reverse().map((item, index) => (
                 <div className="col-md-6">
```
Comment (hansanwok) [inline src/pages/Portfolio/index.js]: where is key?

File: src/pages/Portfolio/index.js [hunk 5]
```diff
@@ -74,47 +79,43 @@ const Services = () => (
...
-                </div>
+                </Link>
+              </div>
               ))}
             </div>
           </Tab>
           <Tab eventKey="mobile" title="Mobile development">
             <div className="row">
               {tabContent.map((item, index) => (
                 <div className="col-md-6">
```
Comment (hansanwok) [inline src/pages/Portfolio/index.js]: where is key?

File: src/utils/portfolioImages.js [hunk 6]
```diff
@@ -0,0 +1,191 @@
+export default {
+  bid4Skill1: "https://drive.google.com/uc?export=view&id=1I9mFdurr5I_Uum-BUPTAbnAG3UISZMNv",
```
Comment (hansanwok) [inline src/utils/portfolioImages.js]: space not is the same

File: src/pages/Portfolio/index.js [hunk 7]
```diff
@@ -74,47 +79,43 @@ const Services = () => (
...
+                      >
+                        {item.content}
+                      </h4>
+                    </div>
+                  </Link>
                 </div>
               ))}
             </div>
           </Tab>
         </Tabs>{" "}
```
Comment (hansanwok) [inline src/pages/Portfolio/index.js]: a space for what, shouldn;t be that

File: src/pages/PortfolioSubPage/index.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+                  </li>
+                ))}
+              </ul>
+            </div>
+            {content.img.map((image, index) => (
+              <div
+                className={`col-md-${image.col} portfolio-details__body__image`}
+                key={`${content.key} ${index}`}
+              >
+                <img src={image.src} alt={`${content.key} ${index}`} loading="lazy" />
```
Comment (hansanwok) [inline src/pages/PortfolioSubPage/index.js]: alt defined for text? why need a key here

File: src/pages/Portfolio/index.js (hunk 1 above)
Comment (austin-pham-goldenowl) [reply src/pages/Portfolio/index.js]: no design yet, so I cloned the array and reversed it to make the second tab look different.

File: src/pages/Portfolio/index.js (hunk 7 above)
Comment (austin-pham-goldenowl) [reply src/pages/Portfolio/index.js]: Prettier document format, my bad.

File: src/pages/PortfolioSubPage/index.js (hunk 8 above)
Comment (austin-pham-goldenowl) [reply src/pages/PortfolioSubPage/index.js]: content.key is the title (project name)

**Pull request merged**